### PR TITLE
remove outdated information

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8472
source: https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/edr-capabilities-for-macos-have-now-arrived/ba-p/1047801

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/threat-protection/microsoft-defender-atp/endpoint-detection-response-mac-preview.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ ms.topic: conceptual
 [!INCLUDE [Microsoft 365 Defender rebranding](../../includes/microsoft-defender.md)]
 
 
-Endpoint detection and response capabilities in Microsoft Defender ATP for Mac are now in preview. To get these and other preview features, you must set up your Mac device to be an "Insider" device as described in this article. For scale deployment, we recommend using [Jamf](#enable-the-insider-program-with-jamf) or [Intune](#enable-the-insider-program-with-intune).
+To get preview features available for Mac, you must set up your device to be an "Insider" device as described in this article. For scale deployment, we recommend using [Jamf](#enable-the-insider-program-with-jamf) or [Intune](#enable-the-insider-program-with-intune).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/endpoint-detection-response-mac-preview.md:27]:
```suggestion
To get preview features for Mac, you must set up your device to be an "Insider" device as described in this article. For scale deployment, we recommend using [Jamf](#enable-the-insider-program-with-jamf) or [Intune](#enable-the-insider-program-with-intune).
```

File: windows/security/threat-protection/microsoft-defender-atp/endpoint-detection-response-mac-preview.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ ms.topic: conceptual
...
 
 >[!IMPORTANT]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/endpoint-detection-response-mac-preview.md:29]:
```suggestion
> [!IMPORTANT]
```

File: windows/security/threat-protection/microsoft-defender-atp/endpoint-detection-response-mac-preview.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ ms.topic: conceptual
...
 >Make sure you have enabled [Microsoft Defender ATP for Mac](microsoft-defender-atp-mac.md#how-to-install-microsoft-defender-atp-for-mac), and pay attention to the “earlyPreview” flag. See documentation for [Jamf](mac-install-with-jamf.md), [Intune](mac-install-with-intune.md) and [manual deployment](mac-install-manually.md) instructions.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/endpoint-detection-response-mac-preview.md:30]:
```suggestion
> Make sure you have enabled [Microsoft Defender ATP for Mac](microsoft-defender-atp-mac.md#how-to-install-microsoft-defender-atp-for-mac), and pay attention to the “earlyPreview” flag. See documentation for [Jamf](mac-install-with-jamf.md), [Intune](mac-install-with-intune.md), and [manual deployment](mac-install-manually.md) instructions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**